BlueHarvest/ProfileFormatter.swift

```diff
@@ -0,0 +1,52 @@
+//
+//  ProfileFormatter.swift
+//  BlueHarvest
+//
+//  Created by David Nutter on 3/19/17.
+//  Copyright © 2017 David Nutter. All rights reserved.
+//
+
+import Foundation
+
+struct ProfileFormatter {
+    
+    struct ProfileKeys {
```

Comment:
Looking at it again, these two structs refer to API JSON keys and Data Persistence keys respectively. The string literals do happen to be identical but that could change (perhaps due to the API). I normally like to keep the data layer and the API layer constants decoupled.